src/components/textarea/textarea.component.tsx

```diff
@@ -198,6 +204,28 @@ export const Textarea = React.forwardRef(
       [ref]
     );
 
+    const [characterCountAriaLive, setCharacterCountAriaLive] = useState<
+      "off" | "polite"
+    >("off");
+
+    // This block of code has been covered in Playwright
+    // istanbul ignore next
+    useEffect(() => {
```

Comment:
**suggestion**: similarly to `Textbox`, what do you think about using `Input`'s `onFocus` and `onBlur` props to update the state, rather than using an effect?

Reply:
See previous comment as that will apply here too.